packages/messenger/Counter/CpuCounter.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Draw\Component\Messenger\Counter;
+
+use Fidry\CpuCoreCounter\CpuCoreCounter;
+use Fidry\CpuCoreCounter\NumberOfCpuCoreNotFound;
+
+class CpuCounter
+{
+    public function count(): int
+    {
+        if (!class_exists(CpuCoreCounter::class) || !class_exists(NumberOfCpuCoreNotFound::class)) {
+            throw new \RuntimeException('"fidry/cpu-core-counter" must be installed');
+        }
+
+        $counter = new CpuCoreCounter();
+
+        try {
+            return $counter->getCount();
```

Comment:
you can do a one liner, try to prevent temporary variable when they are not use elsewhere

return (new  CpuCoreCounter())->getCount();